charts/redis/values.yaml

```diff
@@ -9,3 +9,8 @@ redis:
     port: 6379
   db_name: 1
   expiry: 3600
```

Comment:
We do not need to indent this inside the redis block. Let's keep it outside.

Reply:
It's updated